pandas/tseries/offsets.py

```diff
@@ -1394,7 +1411,7 @@ class WeekOfMonth(_WeekOfMonthMixin, DateOffset):
     _prefix = 'WOM'
     _adjust_dst = True
 
-    def __init__(self, n=1, normalize=False, week=None, weekday=None):
```

Comment:
this doesn't break anything???

Reply:
The default Kmarts raise ATM.

Reply:
where is the test for this? why isn't it raising currently?